lib/rust/ensogl/core/src/application.rs

```diff
@@ -68,6 +73,19 @@ impl Application {
     pub fn new_view<T: View>(&self) -> T {
         self.views.new_view(self)
     }
+
+    /// Set "fake" screen dimensions for unit and integration tests. This is important for a lot of
+    /// position and screen size related computations in the IDE.
+    pub fn set_screen_size_for_tests(&self) {
+        let (screen_width, screen_height) = TEST_SCREEN_SIZE;
+        let scene = &self.display.default_scene;
+        scene.layers.iter_sublayers_and_masks_nested(|layer| {
+            let camera = layer.camera();
+            camera.set_screen(screen_width, screen_height);
+            camera.reset_zoom();
+            camera.update(scene);
+        });
+    }
```

Comment:
this should be in a module which compiles only when test compile. It should not be exposed in normal public API

Reply:
It is not possible if we want to use that method in other crate (integration-test) - the dependencies are never test builds.

Perhaps we could add a "test utilities" module with all test-related implementations (but without `#[cfg(test)]`).

Reply:
Moved it to a separate module with an extension trait.